client/config.ts

```diff
@@ -0,0 +1,6 @@
+export const headerLinks: { name: string; label: string; href: string }[] = [
```

Comment:
Maybe just extract this into an interface

Reply:
but keep it in the same file

Reply:
Done. However, this sounds a bit subjective and should be documented somewhere. Preferably a linter file.